internal/featuretests/kubernetes.go

```diff
@@ -17,12 +17,24 @@ package featuretests
 
 import (
 	v1 "k8s.io/api/core/v1"
+	networking_v1 "k8s.io/api/networking/v1"
 	"k8s.io/api/networking/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-func Backend(svc *v1.Service) *v1beta1.IngressBackend {
+func IngressBackend(svc *v1.Service) *networking_v1.IngressBackend {
```

Comment:
Can we have a TODO here to move everything over to v1 from v1beta1, since v1beta1 will be removed soon?